docs/roadmap/index.md

```diff
@@ -0,0 +1,81 @@
+# Roadmap
+
+!!! hint "Work in Progress"
+    We will be building the roadmap together with the Flux community,
+    our end-users and everyone who is interested in integrating with us.
+    So a lot of this is still TBD - read this as our shopping list of
+    ideas after some brainstorming as Flux maintainers.
+
+## The road to Flux v2
+
+### Flux read-only feature parity
+
+This would be the first stepping stone: we want the GitOps Toolkit to be on-par with today's Flux in
+[read-only mode](https://github.com/fluxcd/flux/blob/master/docs/faq.md#can-i-run-flux-with-readonly-git-access)
+and FluxCloud notifications.
```

Comment:
Maybe add a link to fluxcloud for context?

Reply:
Please add the link so I can merge all your suggestions in one-go.

Reply:
```suggestion
and [FluxCloud](https://github.com/justinbarrick/fluxcloud) notifications.
```